lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -484,6 +484,12 @@ enum LoopControlMask {
     LoopControlDontUnrollMask = 0x00000002,
     LoopControlDependencyInfiniteMask = 0x00000004,
     LoopControlDependencyLengthMask = 0x00000008,
+    LoopControlExtendedControlsMask = 0x80000000,
+};
+
+enum LoopControlExtendedControlsMask {
+  InitiationIntervalINTEL = 5889,
+  MaxConcurrencyLoopINTEL = 5890,
```

Comment:
Are you going to add support of DependencyArrayINTEL ?

Reply:
I will. Later.